proxy/generate_precompiled_config.js

```diff
@@ -231,10 +231,10 @@ const g_arrContracts = [
         address: "0x57ad607c6e90df7d7f158985c3e436007a15d744",
         referenceVariableName: "TokenManagerLinker"
     }, {
-        fileName: "EthERC20.json",
-        filePath: "artifacts/contracts/schain/tokens/EthERC20.sol/EthERC20.json",
+        fileName: "EthErc20.json",
```

Comment:
Do we still need generate precompiled config? If we need this - addresses are incorrect - if not - let's remove

Reply:
Yes. It's duplication. Have removed the file.